tests/Unit/Api/Service/MyParcelApiServiceTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/** @noinspection StaticClosureCanBeUsedInspection,PhpUnhandledExceptionInspection */
+
+declare(strict_types=1);
+
+use Composer\InstalledVersions;
+use MyParcelNL\Pdk\Api\Service\ApiServiceInterface;
+use MyParcelNL\Pdk\Api\Service\MyParcelApiService;
+use MyParcelNL\Pdk\Base\Factory\PdkFactory;
+use MyParcelNL\Pdk\Tests\Bootstrap\MockPdkConfig;
+use function DI\autowire;
+
+it('get correct headers', function () {
```

Comment:
```suggestion
it('gets correct headers', function () {
```

je moet dit lezen als een zin, want dat is ook hoe het eruit ziet als je dit runt:

`✅ it gets correct headers`